collect_app/src/main/java/org/odk/collect/android/permissions/PermissionsProvider.java

```diff
@@ -224,15 +225,14 @@ public void onPermissionRationaleShouldBeShown(List<PermissionRequest> permissio
     }
 
     protected void showAdditionalExplanation(Activity activity, int title, int message, int drawable, @NonNull PermissionListener action) {
-        AlertDialog alertDialog = new AlertDialog.Builder(activity)
-                .setTitle(title)
-                .setMessage(message)
-                .setPositiveButton(android.R.string.ok, (dialogInterface, i) -> action.denied())
-                .setCancelable(false)
```

Comment:
Looks like we lose the cancel blocking here, but I'm not sure that matters. Can you think of any reason we'd want these dialogs to not be cancellable?

Reply:
No I think there is no reason to make it not cancellable.